src/app/_components/ui/tabs.tsx

```diff
@@ -3,12 +3,11 @@ import withForwardRef from '../hocs/with-forward-ref'
 
 const Tabs = TabsPrimitive.Root
 const TabsList = withForwardRef(TabsPrimitive.List, {
-    className:
-        'inline-flex w-full h-[54px] items-center justify-end rounded-full bg-muted text-muted-foreground border-b-2',
+    className: 'inline-flex w-full h-[54px] items-center justify-end rounded-full bg-muted text-muted-foreground',
 })
 const TabsTrigger = withForwardRef(TabsPrimitive.Trigger, {
     className:
-        'inline-flex items-center justify-center h-full text-sm font-medium transition-all data-[state=active]:text-black text-[#B2B2B2] hover:text-zinc-600/70 w-1/2 transparent-tap bg-transparent border-b-2 data-[state=active]:border-black relative -mb-[4px]',
+        'inline-flex items-center justify-center h-full text-sm font-medium transition-all data-[state=active]:text-black text-[#B2B2B2] hover:text-zinc-600/70 w-1/2 transparent-tap bg-transparent relative',
```

Comment:
by changing the styles of the main tabs component, we broke again the styles in the participants list:

Before:
![image](https://github.com/user-attachments/assets/c9a88f3a-3ccb-426a-9afe-699b5379ac22)

Now:
![image](https://github.com/user-attachments/assets/9af4eec8-ddc1-4544-8fc3-db344d9e5e5e)


We should ensure we keep both tabs styles, so I suggest applying the style differences locally on each tabs component (my-pools / participants list)